src/ParticleVaultContainer.cc

```diff
@@ -1,3 +1,6 @@
+
+#include <cstdint>
```

Comment:
Thanks for spotting this problem.  However, this is not the correct fix for this case.  There are two issues:

First, one of our coding rules is that any .cc file should always include the corresponding .hh file first.  This ensures that there are no include files missing from the .hh file.  The real problem you have caught here is that <cstdint> is missing from ParticleVaultContainer.hh.

Second, we've run into issues with other compilers that inexplicably don't have <cstdint>.  To deal with this, we include portability.hh (I know, obvious name, right?).  That file does the right thing when CSTDINT_MISSING is defined.

Hence, the correct solution in this case is to #include "portability.hh" in ParticleVaultContainer.hh